core/src/io/anuke/mindustry/editor/MapRenderer.java

```diff
@@ -97,59 +101,91 @@ public void updateAll(){
     }
 
     private void render(int wx, int wy){
-        int x = wx / chunkSize, y = wy / chunkSize;
-        IndexedRenderer mesh = chunks[x][y];
-        Tile tile = editor.tiles()[wx][wy];
+        final Point windowPoint = new Point(wx, wy);
```

Comment:
You should *never* create objects in a method that may be called hundreds of times per frame. This makes massive amounts of garbage, which is a significant issue for Android phones. Objects should only be created during game initialization or world loading periods.

Even if performance were not a problem, I don't see how using objects here improves readability.